api.Dockerfile

```
@@ -30,4 +30,4 @@ RUN apt-get update -qq \
         libssl-dev
 
 COPY --from=builder /target/release/hawkeye-api .
-ENTRYPOINT ["/hawkeye-api"]
+ENTRYPOINT ["/hawkeye-api start-api"]
```

Interesting, I thought it was necessary to be like:
```suggestion
ENTRYPOINT ["/hawkeye-api", "start-api"]
```

Does this current way also worked?